src/berbl/literal/__init__.py

```diff
@@ -272,9 +293,14 @@ def train_mixing(M: np.ndarray, X: np.ndarray, Y: np.ndarray, Phi: np.ndarray,
 
     Returns
     -------
-    V, Lambda_V_1, a_beta, b_beta : tuple of arrays of shapes (DV, K), (K * DV, K * DV), (K,) and (K,)
-        Mixing weight matrix, mixing weight covariance matrix, mixing weight
-        prior parameter vectors.
+    V : array of shape (DV, K) 
+        Mixing weight matrix
+    Lambda_V_1 : array of shape (K DV, K DV)
```

Comment:
I think we want Python notation in type descriptions, i.e. `K * DV` instead of `K DV`?